### PR TITLE
feat(bottom-button-view): UX

### PR DESCRIPTION
**Pull request by VladislavRiabko**
Снимок экрана 2022-08-30 в 16 42 08

### CODE REVIEW

File: app/components/BottomButtonView/BottomButtonView.props.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { ViewProps } from 'react-native';
+
+export interface IBottomButtonViewProps extends ViewProps {
+  title: string;
+  onPress: () => void;
```
Comment (Reykjavik151) [inline app/components/BottomButtonView/BottomButtonView.props.ts]: rename to `onSubmit`

File: app/components/BottomButtonView/BottomButtonView.styles.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { StyleSheet, ViewStyle } from 'react-native';
+
+export interface IRadioButtonStyles {
+  container: ViewStyle;
+}
+
+export const styles = StyleSheet.create<IRadioButtonStyles>({
+  container: {
+    flex: 1,
+    justifyContent: 'space-between',
```
Comment (Reykjavik151) [inline app/components/BottomButtonView/BottomButtonView.styles.ts]: you don't need this

File: ios/Podfile.lock [hunk 3]
```diff
@@ -9,40 +9,17 @@ PODS:
...
-    - FirebaseCore (~> 9.0)
-    - FirebaseInstallations (~> 9.0)
-    - GoogleAppMeasurement (= 9.5.0)
-    - GoogleUtilities/AppDelegateSwizzler (~> 7.7)
-    - GoogleUtilities/MethodSwizzler (~> 7.7)
-    - GoogleUtilities/Network (~> 7.7)
-    - "GoogleUtilities/NSData+zlib (~> 7.7)"
-    - nanopb (< 2.30910.0, >= 2.30908.0)
-  - FirebaseAuth (9.5.0):
+    - FirebaseAuth (~> 9.4.0)
```
Comment (Reykjavik151) [inline ios/Podfile.lock:14]: agree it with @emberglaz

**Review by Reykjavik151 — CHANGES_REQUESTED** [review]

File: app/components/BottomButtonView/BottomButtonView.styles.ts (hunk 2 above)
Comment (emberglaz) [inline app/components/BottomButtonView/BottomButtonView.styles.ts]: remove

File: app/components/BottomButtonView/BottomButtonView.styles.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { StyleSheet, ViewStyle } from 'react-native';
+
+export interface IRadioButtonStyles {
```
Comment (emberglaz) [inline app/components/BottomButtonView/BottomButtonView.styles.ts]: rename

File: app/components/BottomButtonView/BottomButtonView.props.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import { ViewProps } from 'react-native';
+
+export interface IBottomButtonViewProps extends ViewProps {
+  title: string;
```
Comment (emberglaz) [inline app/components/BottomButtonView/BottomButtonView.props.ts]: `buttonTitle: string`

**Review by emberglaz — CHANGES_REQUESTED** [review]

**Review by emberglaz — APPROVED** [review]

**Review by Reykjavik151 — APPROVED** [review]

**Pull request merged**